cryoet_data_portal_neuroglancer/precompute/segmentation_mask.py

```diff
@@ -441,3 +445,15 @@ def encode_segmentation(
         clean_mesh_folder(output_path, mesh_directory)
 
     LOGGER.info("Wrote segmentation to %s", output_path)
+
+    resolutions_equal = (
```

Comment:
I think 

```python
resolution_equal = resolution == initial_resolution
```
should be enough, unless `resolution` and `initial_resolution` have different sizes.

Reply:
cool, thanks! :)